### PR TITLE
[WIP] update samples

### LINKED ISSUE #15: Code is out of date
As of ReactiveUI v5.0, ReactiveAsyncCommand is no longer available.  I'm brand new to ReactiveUI and looking for best practices before I develop a lot of bad habits.  The principles here are probably still sound, but I can't implement the code as written, and can't be totally sure some that there aren't new best practices for the newer versions.

### PR DESCRIPTION
**Pull request by shiftkey**
Fixes #15 

- [x] Merge #14 and incorporate changes
- [x] Merge #12 and incorporate changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,22 +116,22 @@ public class RepositoryViewModel : ReactiveObject
 {
   public RepositoryViewModel()
   {
-    canDoIt = this.WhenAny(x => x.StuffFetched, y => y.OtherStuffNotBusy, (x, y) => x && y)
+    canDoIt = this.WhenAnyValue(x => x.StuffFetched, y => y.OtherStuffNotBusy, (x, y) => x && y)
```
Comment (kmcginnes) [inline README.md]: This is incorrect. You can use WhenAny or WhenAnyValue here, but WhenAnyValue does not have a selector. Also, the original selector is incorrect. Here are the two correct versions:

```
canDoIt = this.WhenAny(x => x.StuffFetched, y => y.OtherStuffNotBusy, (x, y) => x.Value && y.Value)
```

or

```
canDoIt = this.WhenAnyValue(x => x.StuffFetched, y => y.OtherStuffNotBusy).Select(x => x.Item1 && x.Item2)
```

Comment (shiftkey) [inline README.md]: This compiles against RxUI 6.5.0

screen shot 2015-11-17 at 11 08 23 am

>  Is the `.Value` no longer required for `this.WhenAny`?

Nah, I think this was just me rushing the initial sample...

Comment (kmcginnes) [inline README.md]: Ah, cool. Some new goodies I didn't know about. Thanks.

Comment (shiftkey) [inline README.md]: @kmcginnes  :thumbsup:

**Comment by ghuntley** [comment]
:fire:

Can you please make the changes directly to the documentation repo (or via Gitbooks)

**Pull request closed without merging**